debian/control

```diff
@@ -2,7 +2,7 @@ Source: networking-calico
 Section: net
 Priority: optional
 Maintainer: Neil Jerram <neil@projectcalico.org>
-Build-Depends: debhelper (>= 8.0.0), dh-systemd, python-all (>= 2.6), python-setuptools
+Build-Depends: debhelper (>= 8.0.0), dh-systemd, dh-python, python-all (>= 2.6), python-setuptools
```

Comment:
I'm not 100% sure if we need `dh-python` in this branch for focal builds. I tried testing a build using projectcalico/packaging using branch off this PR with `dh-python` removed and it didn't fail (like Rodrigo saw in #48).

Reply:
Yes I believe that's right.  The `dh-python` addition was related to this `python3` in debian/rules:
```
dh $@ --with python3,systemd --buildsystem=pybuild
```
and this branch doesn't have that; instead it has:
```
dh $@ --with python2,systemd --buildsystem=python_distutils
```
So it makes sense that `dh-python` isn't needed here, and it would be better to revert this change.

Reply:
Thanks @neiljerram !